app/controllers/lists_controller.rb

```diff
@@ -2,4 +2,12 @@ class ListsController < ApplicationController
   def index
     @lists = List.all
   end
+
+  def show
+    @list = List
+      .includes(:books)
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.